sdk/tests/js/tests/ext-float-blend.js

```diff
@@ -29,7 +29,9 @@ void main()
 }
 `;
 
-function testExtFloatBlend(shouldBlend, internalFormat) {
+function testExtFloatBlend(internalFormat) {
+    const shouldBlend = gl.getSupportedExtensions().indexOf('EXT_float_blend') != -1;
```

Comment:
These mean the browser MUST expose EXT_float_blend if it supports float blend, right?

Reply:
That's right.

Reply:
That's very strict, but it is nice. Seems ok with me.

Reply:
SGTM too.